backend/app/mcp/tools.py

```
@@ -0,0 +1,466 @@
+        for s in result.get("sentiments", [])
+    ]
+    trends = _singletons.trend_detector.calculate_weekly_trends(sentiments_with_date)
+    anomalies = _singletons.trend_detector.detect_anomalies(trends)
+
+    summary_data = result.get("summary", {})
+    record = ReviewAnalysis(
+        analysis_type="manual",
+        target_scope=scope,
+        review_count=total_count,
+        sentiment_summary=result.get("sentiment_summary"),
+        keywords=[kw if isinstance(kw, dict) else kw for kw in result.get("keywords", [])],
```

Ruff (0.15.12)

[warning] Useless `if`-`else` condition

(RUF034)